### DISCUSSION TITLE
C# SDK

### DISCUSSION THREAD

**Discussion by PRCV1** [pocketbase/pocketbase#736] [Show and tell]
I love this project and wanted to do some side projects with it. Unfortunately there is no C# SDK, so I started with that. I'm relatively new to git and open source, so forgive me for mistakes.
Likewise, this is the first time I'm trying my hand at "real" unit testing....

The repo is here: https://github.com/PRCV1/pocketbase-csharp-sdk

**Comment by ganigeorgiev** [comment]
Thank you. I don't have experience with C# but I'm sure someone will find it useful!

_I'll move the issue it to "Show and tell" discussion._

**Reply by PRCV1** [reply]
oh nvm, havent seen that there is a discussion tab.
